Example oas file is failing some linter checks

The [example.oas.yaml](https://github.com/aep-dev/aeps/blob/8e34b04ccdfb1c747c2ff0dae87afe56263e881d/aep/general/example.oas.yaml) is failing some of the linter checks (warnings):

```sh
>spectral lint examples/example.oas.yaml    
(node:8508) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)

/Users/Mike/Projects/aep-dev/aep-openapi-linter/examples/example.oas.yaml
  46:18  warning  aep-133-id-parameter  The id parameter is missing.  paths./isbns.post
 116:18  warning  aep-133-id-parameter  The id parameter is missing.  paths./publishers.post
 243:28  warning  aep-133-id-parameter  The id parameter is missing.  paths./publishers/{publisher}/books.post.parameters
 381:28  warning  aep-133-id-parameter  The id parameter is missing.  paths./publishers/{publisher}/books/{book}/editions.post.parameters
```

As these all appear to be standard Create methods I think they should have an `id` parameter.

The new rules for aep-134 (still in development) are also reporting errors:

```text
>spectral lint examples/example.oas.yaml

/Users/Mike/Projects/aep-dev/aep-openapi-linter/examples/example.oas.yaml
 166:29  warning  aep-134-content-type  The request body content type should be "application/merge-patch+json"  paths./publishers/{publisher}.patch.requestBody.content
 298:29  warning  aep-134-content-type  The request body content type should be "application/merge-patch+json"  paths./publishers/{publisher}/books/{book}.patch.requestBody.content

✖ 2 problems (0 errors, 2 warnings, 0 infos, 0 hints)
```

I _think_ we want the request body content type to be "application/merge-patch+json", right?

--- a/aep/general/example.oas.yaml
+++ b/aep/general/example.oas.yaml
@@ -61,6 +61,14 @@
             {
               'description': 'Create method for isbn',
               'operationId': 'CreateIsbn',
+              'parameters':
+                [
+                  {
+                    'name': 'id',
+                    'in': 'query',
+                    'schema': { 'type': 'string' },
+                  },
+                ],
               'responses':
                 {
                   '200':
@@ -137,6 +145,16 @@
                     'in': 'query',
                     'schema': { 'type': 'string' },
                   },
+                  {
+                    'name': 'skip',
+                    'in': 'query',
+                    'schema': { 'type': 'integer' },
+                  },
+                  {
+                    'name': 'filter',
+                    'in': 'query',
+                    'schema': { 'type': 'string' },
+                  },
                 ],
               'responses':
                 {
@@ -171,6 +189,14 @@
             {
               'description': 'Create method for publisher',
               'operationId': 'CreatePublisher',
+              'parameters':
+                [
+                  {
+                    'name': 'id',
+                    'in': 'query',
+                    'schema': { 'type': 'string' },
+                  },
+                ],
               'responses':
                 {
                   '200':
@@ -251,7 +277,7 @@
                       'description': 'Successful response',
                       'content':
                         {
-                          'application/json':
+                          'application/merge-patch+json':
                             {
                               'schema':
                                 { '$ref': '#/components/schemas/publisher' },
@@ -263,7 +289,7 @@
                 {
                   'content':
                     {
-                      'application/json':
+                      'application/merge-patch+json':
                         {
                           'schema':
                             { '$ref': '#/components/schemas/publisher' },
@@ -412,6 +438,11 @@
                     'required': true,
                     'schema': { 'type': 'string' },
                   },
+                  {
+                    'name': 'id',
+                    'in': 'query',
+                    'schema': { 'type': 'string' },
+                  },
                 ],
               'responses':
                 {
@@ -502,7 +533,7 @@
                       'description': 'Successful response',
                       'content':
                         {
-                          'application/json':
+                          'application/merge-patch+json':
                             {
                               'schema':
                                 { '$ref': '#/components/schemas/book' },
@@ -514,7 +545,7 @@
                 {
                   'content':
                     {
-                      'application/json':
+                      'application/merge-patch+json':
                         { 'schema': { '$ref': '#/components/schemas/book' } },
                     },
                   'required': true,
@@ -676,6 +707,11 @@
                     'required': true,
                     'schema': { 'type': 'string' },
                   },
+                  {
+                    'name': 'id',
+                    'in': 'query',
+                    'schema': { 'type': 'string' },
+                  },
                 ],
               'responses':
                 {
@@ -819,14 +855,20 @@
                           'application/json':
                             {
                               'schema':
-                                { '$ref': '#/components/schemas/book' },
+                                {
+                                  'type': 'object',
+                                  'properties':
+                                    { 'success': { 'type': 'boolean' } },
+                                  'x-aep-field-numbers': { '0': 'success' },
+                                },
                             },
                         },
                     },
                 },
               'requestBody':
                 {
-                  'content': { 'application/json': { 'schema': {} } },
+                  'content':
+                    { 'application/json': { 'schema': { 'type': 'object' } } },
                   'required': true,
                 },
             },

--- a/aep/general/example.oas.yaml
+++ b/aep/general/example.oas.yaml
@@ -1,12 +1,13 @@
 {
-  'openapi': '3.1.0',
-  'servers': [{ 'url': 'http://localhost:8081' }],
   'info':
     {
       'title': 'bookstore.example.com',
       'description': 'An API for bookstore.example.com',
       'version': 'version not set',
+      'contact': { 'name': 'API support', 'email': 'aepsupport@aep.dev' },
     },
+  'openapi': '3.1.0',
+  'servers': [{ 'url': 'http://localhost:8081' }],
   'paths':
     {
       '/isbns':

